code/game/objects/items/melee/energy.dm

```diff
@@ -175,14 +175,12 @@
 /obj/item/melee/transforming/energy/sword/cyborg/saw/hit_reaction(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK)
 	return 0
 
-/*
-/obj/item/melee/transforming/energy/sword/cyborg/saw/implant //Energy Saw Arm Implant
-	icon_state = "saw"
-	icon_state_on = "saw"
+/obj/item/melee/transforming/energy/sword/esaw/implant //Energy Saw Arm Implant
+	icon_state = "implant-esaw_0"
+	icon_state_on = "implant-esaw_1"
 	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
```

Comment:
```suggestion
```
this part can go nonetheless since it really is the same as the parent one

Reply:
as a matter of fact, I think i forgot the in-hand sprite for the Implant e-saw, will work on it tomorrow